src/imageio/format/avif.c

```diff
@@ -636,10 +639,8 @@ void *get_params(dt_imageio_module_format_t *self)
       break;
     case AVIF_COMP_LOSSY:
       d->quality = dt_conf_get_int("plugins/imageio/format/avif/quality");
-      if(d->quality > 100)
-      {
-        d->quality = 100;
-      }
+      if(d->quality < 5) d->quality = 5;
```

Comment:
`d->quality = CLAMP(d->quality, 5, 100)`

But even better set the min/max in `darktableconfig.xml.in` and it will get sanitized when loading the values so we can completely remove the check here.

Reply:
> But even better set the min/max in `darktableconfig.xml.in` and it will get sanitized when loading the values so we can completely remove the check here.

Cool, they're already set, so I can just remove all of these checks then!